pymc_experimental/distributions/mvncdf.py

```diff
@@ -22,38 +23,40 @@ def conditional_covariance(Sigma, mu, conditioned, conditioned_values):
     Sigma_12 = Sigma[target][:, conditioned]
     Sigma_11 = Sigma[target][:, target]
 
-    Sigma_cond = Sigma_11 - np.matmul(np.matmul(Sigma_12, np.linalg.inv(Sigma_22)), Sigma_21)
-    mean_cond = np.delete(mu, conditioned) + np.matmul(Sigma_12, np.linalg.inv(Sigma_22)).dot(
-        conditioned_values - mu[conditioned]
-    )
+    inv = pm.math.matrix_inverse(Sigma_22)
+    Sigma_cond = Sigma_11 - pm.math.matrix_dot(Sigma_12, inv, Sigma_21)
+    mean_cond = pm.math.matrix_dot(Sigma_12, inv, conditioned_values)
 
     return Sigma_cond, mean_cond
 
 
 class Mvncdf(Op):
     __props__ = ()
 
-    def make_node(self, upper, mu, cov):
-        upper = at.as_tensor_variable(upper)
+    def make_node(self, value, mu, cov):
+        value = at.as_tensor_variable(value)
         mu = at.as_tensor_variable(mu)
         cov = at.as_tensor_variable(cov)
 
-        return Apply(self, [x], [x.type()])
+        return Apply(self, [value], [value.type()])

```

Comment:
The Apply node specifies all inputs and outputs, so it should be 
```suggestion
        return Apply(self, [value, mu, cov], [value.type()])

```